packages/apps/dashboard/ui/src/constants/index.ts

```diff
@@ -2,12 +2,10 @@ import { ChainId } from '@human-protocol/sdk';
 
 export const V2_SUPPORTED_CHAIN_IDS = [
   ChainId.MAINNET,
-  ChainId.GOERLI,
   ChainId.SEPOLIA,
   ChainId.BSC_MAINNET,
   ChainId.BSC_TESTNET,
   ChainId.POLYGON,
-  ChainId.POLYGON_MUMBAI,
```

Comment:
Adrian mentioned that on a dashboard we still have archive data about testnets. 
Will this change affect it?

Reply:
I think this only removes the Mumbai option from the network selection from the UI. I guess the data itself is coming from the admin API, and if we still have archive data, we should fix backend API. BTW, I don't see those deprecated testnets inside supported chain list from admin API, though.